### PR TITLE
Feature upgrade to Xamarin Mac Universal

### PR DESCRIPTION
**Pull request by Tapped**
By upgrading to Xamarin Mac Universal we'll support 64-bit executables on macOS. 
NOTE: Depends on https://github.com/fusetools/uno/pull/1255 and fusetools/Fuse#4052

This PR contains:
- [ ] Changelog
- [ ] Documentation
- [ ] Tests

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Unreleased
 
+## Video
+- Xamarin Mac was upgraded so that 64-bit executables on macOS will be supported.
+- The `AudioPlayer` property of `VideoHandle` in the Mono CIL backend has been removed.
```
Comment (kusma) [inline CHANGELOG.md]: `VideoHandle` is internal from Unos point of view, so this change shouldn't be user-visible.

**Comment by kusma** [comment]
@mortoray: This depends on a PR in in uno which hasn't been merged yet, it says.

**Comment by mortoray** [comment]
@kusma Sorry about that. Hmm, I wonder if maybe we shouldn't approve PRs until they are actually ready to merge?  I mean, I do review them, but it's apparent I can overlook things.

**Comment by kusma** [comment]
@mortend: I reviewed the code, the approval was only meant to indicate that. But yes, maybe I should have just written it as a comment rather than an approval.

**Pull request merged**